core/templates/dev/head/css/oppia.css

```diff
@@ -3819,16 +3819,26 @@ md-card.preview-conversation-skin-supplemental-card {
   background-color: #fff;
   cursor: pointer;
   display: inline-block;
-  height: inherit;
+  height: fit-content;
```

Comment:
Same question as before; why is this changed?

Reply:
As I was introducing an additional parent div of height and width it would inherit those.

Reply:
OK. Try to find a way to handle that that doesn't modify the CSS of this tile, if possible. Like I said, the tile is used in many other contexts, and if you want to make a change here then you will need to very carefully verify that all of them still work after your change.

Reply:
Took care of it